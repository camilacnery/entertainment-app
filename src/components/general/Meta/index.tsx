import { FC } from "react";
import Head from "next/head";

type TMetaProps = {
  title: string;
  description?: string;
};

const Meta: FC<TMetaProps> = ({ title, description }) => {
  return (
    <Head>
      <title>{title}</title>
      {description && <meta name="description" content={description} />}
      <link rel="icon" href="/favicon.ico" />
    </Head>
  );
};

export default Meta;
