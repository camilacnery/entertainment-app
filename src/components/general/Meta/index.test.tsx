import { render } from "@testing-library/react";

import Meta from ".";

describe("Header", () => {
  it("renders component", () => {
    const { baseElement } = render(<Meta title="Title" description="Description" />);

    expect(baseElement).toMatchSnapshot();
  });
});
