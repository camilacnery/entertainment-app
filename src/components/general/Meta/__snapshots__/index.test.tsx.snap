// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders component 1`] = `
<body>
  <div />
</body>
`;
